test/test_overrides.py

```diff
@@ -363,6 +366,14 @@ def decorator(func):
     torch.range,
     torch.sparse_coo_tensor,
     torch.zeros,
+    torch.nn.functional.assert_int_or_pair,
+    torch.nn.functional.boolean_dispatch,
+    torch.nn.functional.division,
```

Comment:
Do you know why this is here?  AFAICT, the only `division` in torch.nn.functional comes from `from __future__ import division`, which is probably not what you want.

Reply:
It's a blacklist: ```IGNORED_TORCH_FUNCTIONS```. The test loops over `dir` of the module and somehow division was being exported lol